apps/frontend/src/pages/onboarding/libs/components/onboarding-answer/onboarding-answer.tsx

```diff
@@ -11,7 +12,10 @@ const OnboardingAnswer = <T extends FieldValues>({
 	control,
 	name,
 	options,
-}: Properties<T>): JSX.Element => {
+}: { options: InputOption[] } & Omit<
+	Properties<T>,
+	"options"
+>): JSX.Element => {
```

Comment:
Why do you use Omit and add options again?

Reply:
Because the `FormFieldProperties` type had nullable `options`, but that component require option.

Should I create its own type like `QuizQuestion` component does?

```ts
type FormFieldProperties<T extends FieldValues> = {
	control: Control<T, null>;
	name: FieldPath<T>;
	options?: InputOption[];
};

type QuizQuestionPropeties<T extends FieldValues> = {
	control: Control<T, null>;
	label: string;
	name: FieldPath<T>;
	options: InputOption[];
};
```

Reply:
Or I will just do

```ts
import { Input } from "~/libs/components/components.js";
import {
	type Control,
	type FieldPath,
	type FieldValues,
	type InputOption,
} from "~/libs/types/types.js";

import styles from "./styles.module.css";

type Properties<T extends FieldValues> = {
	control: Control<T, null>;
	name: FieldPath<T>;
	options: InputOption[];
};

const OnboardingAnswer = <T extends FieldValues>({
	control,
	name,
	options,
}: Properties<T>): JSX.Element { ... }
```